DetailsPopup.js

```diff
@@ -0,0 +1,185 @@
+    Skills += `<li>${skill}</li>`;
+  });
+  const card = `
+    <section class="card flexbox-column">
+      <div class="card-box">
+        <img
+          src="${project.featuredImage}"
+          alt="Tonic Company"
+          class="tonic-image"
+        />
+      </div>
+      <div class="card-box">
+        <h1 class="TonicHeader">${project.name}</h1>
+        <div class="HeaderTitle flexbox-row_socialicon">
+          <h2 class="tonic-title" style="leftmarign=30px">${project.projectInfo[0]}</h2>
+          <img src="Asset/Counter.png" class="counter" />
+          <h3 class="TonicEnd">${project.projectInfo[1]}</h3>
+          <img src="Asset/Counter.png" class="counter" />
+          <h3 class="TonicEnd">${project.projectInfo[2]}</h3>
+        </div>
+        <p class="Content">
+          ${project.description}
+        </p>
+        <ul class="skill-box flexbox-row_socialicon">
+          ${Skills}
+        </ul>
+        <input type="submit" class="btnResume" value="See Project" id="see-project-${index}">
+      </div>
+    </section>`;
+  template += card;
+});
+projectElement.innerHTML = template;
+
+// POPUP window
+
+const popupContainer = document.createElement('div');
+popupContainer.classList.add('popup-container');
+
+const overlay = document.querySelector('#popup');
+const seeProjectEl = document.querySelectorAll("[id^='see-project-']");
+
+function createPopup(project) {
+  let Skills = '';
+  project.Skills.forEach((skill) => {
+    Skills += `<li>${skill}</li>`;
+  });
+  const popupWindows = `
+      <div class="popup">
+        <div class="project-card Project-container container-popup">
+          <div class="project-title">
+            <h3 class="heading-project">${project.name}</h3>
+            <button class="popup-close-btn">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="popup-close-icon">
+              <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12"></path>
+            </svg>
+          </button>
+          </div>
+          <div class="HeaderTitle flexbox-row_socialicon">
+            <h2 class="tonic-title">${project.projectInfo[0]}</h2>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">${project.projectInfo[1]}</h3>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">${project.projectInfo[2]}</h3>
+          </div>
+          <div class="overflow">
+            <img class="project-img popup-img tonic-image" src="${project.featuredImage}" alt="" />
+          </div>
+          <div class="project-detail popup-details">
+            <p class="PopProjectdescription popup-description">
+              ${project.description}
+            </p>
+            <div class="popup-Skills">
+              <ul class="skill-box flexbox-row_socialicon">
+                ${Skills}
+              </ul>
```

Comment:
- [ ] On desktop screens, kindly ensure that that the technologies unordered list is aligned with the buttons. Kindly see the attached images below 👇 
## Your Design
![image](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/46839548/9f9fcc7c-441e-4647-9cd3-51753e0ab3a2)
## Figma Design
![image](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/46839548/15a9cce5-3661-4b9d-a4df-df2e819b5b17)